test/functional/autocmd/filetype_spec.lua

```diff
@@ -0,0 +1,17 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local eval = helpers.eval
+local clear = helpers.clear
+local command = helpers.command
+
+describe('autocmd FileType', function()
+    before_each(clear)
+
+    it("is triggered by :help only once", function()
+        local exp = 1
+        command("let g:foo = 0")
+        command("autocmd FileType help let g:foo = g:foo + 1")
+        command("help help")
+        assert.same(exp, eval('g:foo'))
```

Comment:
can just have 1 here, no need for `exp` variable.

Reply:
OK, sure